sdk/eventgrid/tests.data.yml

```diff
@@ -0,0 +1,11 @@
+trigger: none
```

Comment:
You didn't add a tests.mgmt.yml does that mean you don't want live test runs for the mgmt library?

Reply:
Yes, I think it isn't necessary to run live test for mgmt. plane sdk

Reply:
Is that true for all the mgmt plane libraries? Or just this one? Do we ever plan to run live tests for the mgmt libraries?

Reply:
Yes, the mgmt code is auto-gened from swagger, so mostly it's not worth to run regular live testing because actually the tests are testing service not test sdk.
We only got ask from ASH team which want to have live test.

Reply:
OK. What is the process for creating and updating the recordings for the tests then?

Reply:
Required to create / update the recordings before release a stable package